libcnmc/res_4771/POS.py

```diff
@@ -40,6 +40,17 @@ def get_sequence(self):
         return self.connection.GiscedataCtsSubestacionsPosicio.search(
             search_params, 0, 0, False, {'active_test': False})
 
+    def get_description(self, sub_id):
+        o = self.connection
+        sub = o.GiscedataCtsSubestacionsPosicio.read(sub_id, ['subestacio_id'])
+        descripcio = ''
+        if sub:
+            ct_id = sub['subestacio_id'][0]
+            ct = o.GiscedataCts.read(ct_id, ['descripcio'])
```

Comment:
subestacio_id is an id from `giscedata.cts.subestacions` not from `giscedata.cts`.